CONTRIBUTING.adoc

```diff
@@ -0,0 +1,184 @@
+and to consolide items more easily during issue triage.
+////
+
+Knative uses Github issues to track documentation issues and requests. If you see a problem with the documentation that you’re not sure how to fix, submit an issue using the following steps:
+
+. Check the https://github.com/knative/docs/issues[Knative docs issues list] before creating an issue to avoid making a duplicate.
+. Use the https://github.com/knative/docs/issues/new[correct template] for your new issue. There are two templates available:
+* *Bug report*: If you’re reporting an error in the existing documentation, use this template. This could be anything from broken samples to typos. When you create a bug report, include as many details as possible and include suggested fixes to the issue. If you know which Knative component your bug is related to, you can assign the appropriate https://github.com/knative/community/blob/master/working-groups/WORKING-GROUPS.md[Working Group Lead].
+* *Feature request*: For upcoming changes to the documentation or requests for more information on a particular subject.
+
+Note that code issues should be filed against the http://github.com/knative[individual Knative repositories], while documentation issues should go in the https://github.com/knative/docs/issues[`knative/docs` repo]. If the issue is specific to the https://knative.dev website, open the issue in the https://github.com/knative/website/issues[`knative/website` repo].
+
+
+=== Fixing Issues (Pull Requests)
+
+////
+This could use a pass to be more focused on what a PR submitter should do at the start of the process.
+////
+
+Here’s what generally happens when you open a PR against the `knative/docs` repo:
+
+. One of the assigned repo maintainers will triage the PR by assigning relative priority, adding appropriate labels, and performing an initial documentation review.
+. If the PR involves significant technical changes, for example new features, or new and changed sample code, the PR is assigned to a Subject Matter Expert (SME), typically an engineer working on Knative, for technical review and approval.
+. When both the technical writers and SMEs are satisfied with the quality of the writing and the technical accuracy of the content, the PR can be merged. A PR requires two labels before it can merge: `lgtm` and `approved`.
+* The SME is responsible for reviewing the technical accuracy and adding the `lgtm` label.
+* The https://github.com/knative/docs/blob/master/OWNERS_ALIASES[Knative technical writers] are who provide the `approved` label when the content meets quality, clarity, and organization standards (see link:#style-guide[Style Guide]).
+
+We appreciate contributions to the docs, so if you open a PR we will help you get it merged. You can also post in the `#docs` https://knative.slack.com/[Slack channel] to get input on your ideas or find areas to contribute before creating a PR.
+
+== Getting involved
```

Comment:
Should this be nearer the top?

Reply:
That being said, I really like this section

Reply:
I can swap it with reporting issues, but the CoC and CLA parts seem like they probably belong before this section, since those will be roadblocks to getting a PR submitted.

Reply:
lol, I thought this was referring to "sending a PR" vs "filing an issue".

See my comment on the PR that there are 3 purposes to this doc:

1. How-To doc for quick hit-and-run PRs / issues from people familiar with GitHub
2. Tutorial on how to contribute for newbies, possibly without GitHub experience
3. Reference documentation for seasoned contributors

This is the start of the "tutorial" section. Other ideas for structuring the content are welcome, but I expect that "group 1" will look at the TOC and first 3-4 paragraphs in `CONTRIBUTING.md` and jump if they don't see what they are looking for.